test/fixtures/invite.fixtures.js

```diff
@@ -49,12 +49,6 @@ module.exports = {
     return opts.map(buildInviteWithDefaults)
   },
 
-  notPermittedInviteResponse: (userName, serviceId) => {
```

Comment:
I assume this was already completely unused.

Reply:
Yes